modules/free/src/main/scala/doobie/free/kleisliinterpreter.scala

```diff
@@ -125,6 +125,9 @@ class KleisliInterpreter[M[_]](logHandler: LogHandlerM[M])(implicit val asyncM:
   def fromFuture[G[_], J, A](interpreter: G ~> Kleisli[M, J, *])(fut: Free[G, Future[A]]): Kleisli[M, J, A] = Kleisli(j =>
     asyncM.fromFuture(fut.foldMap(interpreter).run(j))
   )
+  def fromFutureCancelable[G[_], J, A](interpreter: G ~> Kleisli[M, J, *])(fut: Free[G, (Future[A], Free[G, Unit])]): Kleisli[M, J, A] = Kleisli(j =>
```

Comment:
For a reviewer: As far as I know this (and the same one in the postgres module) is the only "interesting" details outside WeakAsync/WeakAsyncSuite (which is just based on fromFuture).